src/libraries/CashWrapper.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/trading/ICash.sol';
+import 'ROOT/Controlled.sol';
+import 'ROOT/Augur.sol';
+
+
+/**
+ * @title Provides modifiers which take care of Cash/ETH conversion
+ */
+contract CashWrapper is Controlled {
```

Comment:
Naming: This contract doesn't actually wrap cash (which the name implies), it makes modifiers available that allow derived contracts to auto-wrap ETH with cash.  I wonder if we can come up with a name that makes that more clear?

Maybe `CashAutoConvert`?  `Apple is CashAutoConverter`?

Reply:
`CashAutoConverter` sounds good to me